spec/integration/new_spec.rb

```diff
@@ -268,25 +268,19 @@ def exec_exist?(*args)
     end
   end
 
-  context "when app directory contains a whitespace" do
-    subject(:create_app) { Open3.capture3(command) }
+  it "does not raise errors if app directory contains whitespace" do
+    app_directory = "weird dir"
+    FileUtils.mkdir_p("./tmp/#{app_directory}")
```

Comment:
In the `spec_helper` file there are various path manipulation helpers. You don't have to use `FileUtils` and it would be good to avoid hard coding any directory names like `tmp`. So in this case I would do: `app_path = tmp_path("weird dir with spaces")`

Reply:
In ordeer to reproduce the bug specified in the issue I had to use `Dir.mkdir` to create a directory with whitespace. There was no suited helper for this so i inlined it. Hope it's ok 😄

Reply:
What about `dir_path`? 😉